core/include/drivers/rtc.h

```diff
@@ -0,0 +1,141 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2022 Microchip.
+ */
+
+#ifndef DRIVERS_RTC_H
+#define DRIVERS_RTC_H
+
+#include <tee_api_types.h>
+#include <util.h>
+
+/* The RTC allows to set/get offset for correction */
+#define RTC_CORRECTION_FEATURE	BIT(0)
+
+#define TIME_INIT(year, month, day, hour, min, sec) \
```

Comment:
This macro seems a bit unnecessary. If you don't like the order in the struct you can change it there instead.

Reply:
Yes, you are right, it was used previously and was meant to generate a timestamp rather than the struct. Will remove.